cmd/trace-agent/config/config.go

```diff
@@ -285,6 +285,7 @@ func applyDatadogConfig(c *config.AgentConfig) error {
 		BindHost:               c.ReceiverHost,
 		GRPCPort:               grpcPort,
 		MaxRequestBytes:        c.MaxRequestBytes,
+		MaxGrpcRequestMib:      coreconfig.Datadog.GetFloat64("otlp_config.receiver.protocols.grpc.max_recv_msg_size_mib"),
```

Comment:
I don't get why we need this. Otherwise it looks good to me :)

Reply:
I tested more without this and it seems to work fine, so going to remove this